.github/workflows/build-publish-develop-pr.yml

```diff
@@ -113,45 +105,52 @@ jobs:
 
       - name: Build images for ${{ matrix.goarch }}
         uses: ./.github/actions/goreleaser-build-sign-publish
-        if: steps.cache.outputs.cache-hit != 'true'
+        if: github.event_name == 'workflow_dispatch' || steps.cache.outputs.cache-hit != 'true'
         with:
           docker-registry: ${{ secrets.AWS_SDLC_ECR_HOSTNAME }}
           docker-image-tag: ${{ needs.image-tag.outputs.image-tag }}
           goreleaser-release-type: ${{ needs.image-tag.outputs.release-type }}
           goreleaser-config: .goreleaser.develop.yaml
           goreleaser-key: ${{ secrets.GORELEASER_KEY }}
 
-  image-tag:
+  merge:
     runs-on: ubuntu-latest
-    outputs:
-      image-tag: ${{ steps.get-image-tag.outputs.image-tag }}
-      release-type: ${{ steps.get-image-tag.outputs.release-type }}
+    needs: [split, image-tag]
+    if: ${{ needs.image-tag.outputs.release-type == 'nightly' }}
```

Comment:
will this not kick off manually then?

Reply:
`nightly` here is an [option in GoReleaser](https://goreleaser.com/customization/nightlies/) to build and publish the Docker image (not just something that runs 'daily', confusingly). This option should be set when kicking off manually from here: https://github.com/smartcontractkit/chainlink/pull/15480/files#diff-77111379631a187551652b5f718ecfd1ca499e540a41d6d5b6c2e7f651e588c2R51-R56